CONTRIBUTING.md

```diff
@@ -1,12 +1,12 @@
 Here are some basic things to keep in mind
 
-## ARCSlots design goals
-* ARCSlots should be as user friendly as possible
-* ARCSlots should be as configurable as possible _in-game_ without the need for configuration files:
+## ARCPhone design goals
+* ARCPhone should be as user friendly as possible
+* ARCPhone should be as configurable as possible _in-game_ without the need for configuration files:
 
 ## Things that I've always wanted to do but never got around to it.
 * **APP API NEEDS AN OVERHAULED AND MUST BE DOCUMENTED**
-* Make the addon compatible with the ARCLib translation system
+* ~~Make the addon compatible with the ARCLib translation system~~
```

Comment:
I wouldn't call this complete just yet

Reply:
I will try to find and setup any string i find for the translation system asap